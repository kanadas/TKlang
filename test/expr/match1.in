f x = match x with
    1 @ n -> "int";
    2 @ c -> "char";
