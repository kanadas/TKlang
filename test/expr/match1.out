expr/match1.in

Parse              Failed...

syntax error at line 1, column 5 before `='
